worthapp/css/worth.css

```diff
@@ -1,112 +1,103 @@
 @media only screen and (min--moz-device-pixel-ratio: 1.5),
 only screen and (-o-min-device-pixel-ratio: 3/2),
 only screen and (-webkit-min-device-pixel-ratio: 1.5),
-only screen and (min-device-pixel-ratio: 1.5) {
-  body {
-    font-size: 150%;
-  }
-  body button.btn {
-    font-size: 150%;
-  }
+only screen and (min-device-pixel-ratio: 1.5)
+{
+	body { font-size: 150%; }
```

Comment:
we use 4 spaces instead of tabs as indentation everywhere - maybe we should set up some tool to automatically check this for CSS